File: Prepare/Tutorials/30 Days of Code/Python 3/Day 8 - Dictionaries and Maps.py
# Enter your code here. Read input from STDIN. Print output to STDOUT

n = int(input())
name_numbers = [input().split() for _ in range(n)]
phone_book = {k: v for k, v in name_numbers}

while True:
    try:
        searchInput = input()

        if searchInput in phone_book:
            print(f"{searchInput}={phone_book[searchInput]}")
        else:
            print("Not found")
    except EOFError:
        break

File: Prepare/Tutorials/30 Days of Code/C#/Day 8 - Dictionaries and Maps.cs
class Solution
{
    static void Main(string[] args)
    {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
        int numberOfEntries = Convert.ToInt32(Console.ReadLine());
        Dictionary<string, string> phoneBook = new Dictionary<string, string>();

        for (int i = 1; i <= numberOfEntries; i++)
        {
            string[] phoneBook_temp = Console.ReadLine().Split(' ');
            phoneBook[phoneBook_temp[0]] = phoneBook_temp[1];
        }

        string? userInput;
        while ((userInput = Console.ReadLine()) != null)
        {
            try
            {
                Console.WriteLine(userInput + "=" + phoneBook[userInput]);
            }
            catch
            {
                Console.WriteLine("Not found");
            }
        }
    }
}